resources/build/xcode-utils.sh

```diff
@@ -3,11 +3,26 @@
 # This script will automatically have Xcode's build environment (and variables),
 # so there's no need to do anything extra to fetch them.
 
+#############################################################
 # Some fun details for interacting with Xcode:
 #  - echo "warning: foo" will generate an actual compile warning within Xcode:  "foo"
 #  - echo "error: bar" will likewise generate a compile error within Xcode: "bar"
 #
 # As appropriate, make sure to take advantage of this feature!
+##############################################################
+
+# Imports the autogenerated environment.sh from command-line builds.
+function importEnvironment() {
+  # Requires that KEYMAN_ROOT is set... which it should be, if a build is successfully 
+  if [ -z $KEYMAN_ROOT ]; then
+    echo "error: KEYMAN_ROOT is not defined.  Recommendation:  define it as a project-wide \"user-defined\" Build Setting for the \"$PROJECT_NAME\" project."
```

Comment:
Can we define a function `writeError` or `echoError` and corresponding `echoWarning` that encapsulates this? I know it's a bit 'simple' but helps to clarify usage.